api/v1alpha1/flowcollector_types.go

```diff
@@ -482,7 +495,7 @@ type FlowCollectorStatus struct {
 //+kubebuilder:object:root=true
 //+kubebuilder:subresource:status
 //+kubebuilder:resource:scope=Cluster
-//+kubebuilder:printcolumn:name="Agent",type="string",JSONPath=`.spec.agent`
+//+kubebuilder:printcolumn:name="Agent",type="string",JSONPath=`.spec.agent.type`
```

Comment:
Do you know if, with discrimated union, there is a way to add a "sampling" column that shows either ebpf sampling or ipfix sampling?
I wanted to do that before, but couldn't find a way.

Reply:
Hmmm i don't know the way. Even if they are name "union", at the end they follow different paths: `.spec.agent.ipfix` and `.spec.agent.ebpf`